### PR TITLE
Specify GatherND-5 operation

### PR DESCRIPTION
**Pull request by rkazants**
Signed-off-by: Roman Kazantsev <roman.kazantsev@intel.com>

### CODE REVIEW

File: docs/ops/movement/GatherND_5.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+**Outputs**:
+
+*   **1**: Tensor with gathered values of type *T*.
+
+**Types**
+
+* *T*: any supported type.
+
+* *T_IND*: any supported integer types.
+
```
Comment (rkazants) [inline docs/ops/movement/GatherND_5.md:125]: int32, int64 types

**Comment by AndrewBakalinIntel** [comment]
@rkazants is there any specification from the original framework to compare it with?

**Comment by rkazants** [comment]
> @rkazants is there any specification from the original framework to compare it with?

https://www.tensorflow.org/api_docs/java/org/tensorflow/op/core/GatherNd
https://github.com/onnx/onnx/blob/master/docs/Operators.md#GatherND
https://mxnet.apache.org/versions/1.6/api/r/docs/api/mx.nd.gather.nd.html

it is missed in Pytorch

File: docs/ops/movement/GatherND_5.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+`indices` is `K`-dimensional integer tensor or `K-1`-dimensional tensor of tuples with indices by which the operation gathers elements or slices
+from `data` tensor. A position `i_0, ..., i_{K-2}` in the `indices` tensor corresponds to a tuple with indices `indices[i_0, ..., i_{K-2}]`
+of a length equal to `indices.shape[-1]`. By this tuple with indices the operation gathers a slice or an element from `data` tensor and
+insert it into the output at position `i_0, ..., i_{K-2}` as the following formula:
+
+output[i_0, ..., i_{K-2},:,...,:] = data[indices[i_0, ..., i_{K-2}],:,...,:]
+
+The last dimension of `indices` tensor must be not greater than a rank of `data` tensor, i.e. `indices.shape[-1] <= data.rank`.
+The shape of the output can be computed as `indices.shape[:-1] + params.shape[indices.shape[-1]:]`.
```
Comment (AndrewBakalinIntel) [inline docs/ops/movement/GatherND_5.md:19]: `data.shape[indices.shape[-1]:]`

**Comment by dmitry-gorokhov** [comment]
@nshchego Please take a look

File: docs/ops/movement/GatherND_5.md (hunk 2 above)
Comment (rkazants) [reply docs/ops/movement/GatherND_5.md]: Fixed.

**Review by AndrewBakalinIntel — APPROVED** [review]

**Review by nshchego — APPROVED** [review]

**Review by postrational — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**